pkgs/development/libraries/graphene/default.nix

```diff
@@ -11,18 +11,22 @@
 , mutest
 , nixosTests
 , glib
+, withDocumentation ? !stdenv.hostPlatform.isStatic
 , gtk-doc
 , docbook_xsl
 , docbook_xml_dtd_43
+, buildPackages
 , gobject-introspection
+, withIntrospection ? lib.meta.availableOn stdenv.hostPlatform gobject-introspection && stdenv.hostPlatform.emulatorAvailable buildPackages
 , makeWrapper
 }:
 
 stdenv.mkDerivation rec {
   pname = "graphene";
   version = "1.10.8";
 
-  outputs = [ "out" "dev" "devdoc" ]
+  outputs = [ "out" "dev" ]
+    ++ lib.optionals withDocumentation [ "devdoc" ]
     ++ lib.optionals (stdenv.hostPlatform == stdenv.buildPlatform) [ "installedTests" ];
```

Comment:
Maybe make this also dependent on `withIntrospection` since the installed tests will be broken without the wrapping.

Reply:
I ran them with introspection disabled and they seem fine?

Reply:
Are you running `pkgsStatic.graphene.tests.installedTests`? That fails for me due to systemd being marked as broken.

Or manually, with

```bash
nix-shell -I nixpkgs=$PWD -p gnome-desktop-testing --run 'gnome-desktop-testing-runner --dir=$(nix-build -A pkgsStatic.graphene.installedTests)/share'
```

Actually, `pkgsStatic.graphene` does not even appear to have `installedTests` output for me on this branch.

Reply:
I did it manually by modifying nixos/tests/installed-tests/graphene.nix to disable introspection.

Reply:
Ah, looks like they actually do not install the test that relies on introspection when it is disabled.